browser/src/control/Control.JSDialogBuilder.js

```diff
@@ -1082,7 +1082,7 @@ L.Control.JSDialogBuilder = L.Control.extend({
 			$(frameChildren).addClass('expanded');
 
 			var children = [];
-			for (var i = 1; i < data.children.length; i++) {
+			for (var i = 0; i < data.children.length; i++) {
```

Comment:
0 was label, and above code already added that label into the DOM, don't we get duplicated frame title in some cases?